### PR TITLE
0507 rel notes

### PR DESCRIPTION
**Pull request by kaarthis**
Please review team.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+### Release notes
+
+* Preview Features
+   * Mariner is now supported in [NodeOSUpgradeChannel (preview)](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image). This change is currently rolling out and expected to be in all regions by May 12th.
+
+* Bug Fixes
+   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
+   * Now returning a clientError, when you do Stop/RunCommand action on a cluster that was never successfully provisioned and was stuck in failed state.
+   * Upgraded blob csi driver to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: this should be under component updates section

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,33 @@
...
+
+### Release notes
+
+* Preview Features
+   * Mariner is now supported in [NodeOSUpgradeChannel (preview)](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image). This change is currently rolling out and expected to be in all regions by May 12th.
+
+* Bug Fixes
+   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
+   * Now returning a clientError, when you do Stop/RunCommand action on a cluster that was never successfully provisioned and was stuck in failed state.
```
Comment (shashankbarsin) [inline CHANGELOG.md:20]: do we have the exact error message?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,33 @@
...
+   * Upgraded blob csi driver to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
+
+* Behavior Changes
+   * The maximum size of a request body for Nginx ingress controller default increased from 1MB to 64MB.Refer ARM specs [here](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#max-request-body-size).
+   * The Docker Bridge CIDR field in the AKS API was made redundant during our change from Docker to containerD in Kubernetes version 1.19. Starting with the 2023-04-01 AKS API version, the Docker Bridge CIDR field will be removed.
```
Comment (shashankbarsin) [inline CHANGELOG.md:24]: should this be under announcements?

Comment (kaarthis) [reply CHANGELOG.md:24]: Good point.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+* Bug Fixes
+   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
+   * Now returning a clientError, when you do a Stop/RunCommand action on a cluster that was never successfully provisioned and was stuck in failed state. Error message returned is "The cluster is being deleted or hasn't been fully provisioned yet.".
+   * Upgraded blob csi driver to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
+
+* Component Updates
+  * AKS updated the OSM addon on clusters running AKS k8s version >= v1.24.0 from OSM v1.2.3 to v.1.2.4. Details [here](https://github.com/openservicemesh/osm/releases/tag/v1.2.4).
+  * Azure Service Mesh add-on's Istiod and ingress images updated from v1.17.1 to [v1.17.2 patch](https://github.com/istio/istio/compare/1.17.1...1.17.2).
```
Comment (shashankbarsin) [inline CHANGELOG.md:26]:
```suggestion
  * Istio-based service mesh add-on's istiod and ingress images updated from v1.17.1 to v1.17.2. User needs to restart the workload pods to trigger re-injection of the newer patch version of istio-proxy. More information can be found [here](https://learn.microsoft.com/azure/aks/istio-upgrade).
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,30 @@
 # Azure Kubernetes Service Changelog
+## Release 2023-05-07
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+ 
+* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
```
Comment (shashankbarsin) [inline CHANGELOG.md:8]: We can remove this line. It's now more than a month

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,30 @@
...
+
+* Bug Fixes
+   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
+   * Now returning a clientError, when you do a Stop/RunCommand action on a cluster that was never successfully provisioned and was stuck in failed state. Error message returned is "The cluster is being deleted or hasn't been fully provisioned yet.".
+   * Upgraded blob csi driver to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
+
+* Component Updates
+  * AKS updated the OSM addon on clusters running AKS k8s version >= v1.24.0 from OSM v1.2.3 to v.1.2.4. Details [here](https://github.com/openservicemesh/osm/releases/tag/v1.2.4).
```
Comment (shashankbarsin) [inline CHANGELOG.md:25]:
```suggestion
  * Open Service Mesh add-on images updated from v1.2.3 to [v1.2.4](https://github.com/openservicemesh/osm/releases/tag/v1.2.4) for AKS clusters of version >= 1.24.0.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,30 @@
...
+  * Azure Service Mesh add-on's Istiod and ingress images updated from v1.17.1 to [v1.17.2 patch](https://github.com/istio/istio/compare/1.17.1...1.17.2).
+  * Upgraded Cilium to version 1.12.8 along with changes to match [upstream](https://github.com/cilium/cilium/pull/24075/files#diff-8d03af2c9f8f368b86831ae81cfb6f4152e884baf74366c09f2d92363ad791e7).
```
Comment (shashankbarsin) [inline CHANGELOG.md:27]:
```suggestion
  * Cilium upgraded to [1.12.8](https://github.com/cilium/cilium/releases/tag/v1.12.8) for [AKS clusters with Azure CNI Powered by Cilium](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium).
```

**Comment by qpetraroia** [comment]
Signing off given that @shashankbarsin's suggestions are committed

**Review by qpetraroia — APPROVED** [review]

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -16,15 +15,17 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
    * Mariner is now supported in [NodeOSUpgradeChannel (preview)](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image). This change is currently rolling out and expected to be in all regions by May 12th.
 
 * Bug Fixes
-   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Csi-provisioner version updated to v3.5.0 in order to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
    * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
```
Comment (shashankbarsin) [inline CHANGELOG.md:19]: Updated 'cilium', 'cilium-operator', 'cilium-pre-flight' ClusterRoles to include 'update' permission for 'ciliumidentities' api-resource. This addresses the issue where cilium-operator and cilium-agent couldn't garbage collect unused identities.

File: CHANGELOG.md [hunk 9]
```diff
@@ -16,15 +15,17 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
    * Mariner is now supported in [NodeOSUpgradeChannel (preview)](https://learn.microsoft.com/azure/aks/auto-upgrade-node-image). This change is currently rolling out and expected to be in all regions by May 12th.
 
 * Bug Fixes
-   * Upgraded csi-provisioner version to v3.5.0 to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+   * Csi-provisioner version updated to v3.5.0 in order to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
```
Comment (shashankbarsin) [inline CHANGELOG.md:18]: this is component version bump. we need to move it there

File: CHANGELOG.md [hunk 10]
```diff
@@ -16,15 +15,17 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
...
    * Added update permission for ciliumidentity, without this permission- cilium-operator and ciliumAgent cannot garbage collect unused [identities](https://github.com/cilium/cilium/commit/2adf5f4313d42ae055528b09eb8dff4c19e74a98).
    * Now returning a clientError, when you do a Stop/RunCommand action on a cluster that was never successfully provisioned and was stuck in failed state. Error message returned is "The cluster is being deleted or hasn't been fully provisioned yet.".
-   * Upgraded blob csi driver to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
+   * The CPU limit of Azure Monitor Metrics Addon (Windows Daemonset) is updated from 200m to 500m to fix a throttling issue.
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]: The CPU limit Windows DaemonSet ofAzure Monitor Metrics Addon is updated from 200m to 500m to fix throttling issue.

File: CHANGELOG.md [hunk 11]
```diff
@@ -5,7 +5,6 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 
 ### Announcements
  
-* Starting on March 21, 2023, traffic to k8s.gcr.io will be redirected to registry.k8s.io, following the [community announcement](https://kubernetes.io/blog/2023/03/10/image-registry-redirect/).
 * Docker container runtime will be retired for Windows nodepools on May 1, 2023. After docker container runtime is retired, you may remain on existing deployed instances but scaling operations will fail, nodepool creation will fail, and you will be out of support. Follow the detailed steps [in our documentation](https://learn.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli) to upgrade to containerd.
```
Comment (shashankbarsin) [inline CHANGELOG.md:8]: nit for tense: "Docker container runtime will be retired for Windows nodepools on May 1, 2023" -> "Docker container runtime has been retired for Windows nodepools on May 1, 2023"

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,4 +1,31 @@
...
+  * Cilium upgraded to [1.12.8](https://github.com/cilium/cilium/releases/tag/v1.12.8) for [AKS clusters with Azure CNI Powered by Cilium](https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium).
+  * Blob csi driver upgraded to [v1.19.5](https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.19.5) on AKS 1.24, 1.25 to fix blobfuse install failures.
+  * Csi-provisioner version updated to v3.5.0 in order to fix a volume deletion issue, [details](https://github.com/kubernetes/kubernetes/issues/100485#issuecomment-1497878875)
+  * Validations logic added to error out in [Azure CNI overlay](https://learn.microsoft.com/en-us/troubleshoot/azure/azure-kubernetes/insufficientsubnetsize-error-advanced-networking) clusters, whenever autoscaler max count or manual scale node count likely goes over the IP limits of what the managed subnet and podcidr settings get by default
```
Comment (palma21) [inline CHANGELOG.md:28]: Why is the validation a component update and not a bug fix or behavioral change?

Comment (palma21) [reply CHANGELOG.md:28]: > whenever autoscaler max count or manual scale node count likely goes over the IP limits of what the managed subnet and podcidr settings get by default

also what does this mean? What happened before that now doesn't happen or vice versa?

**Review by palma21 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,4 +1,31 @@
...
```
Comment (palma21) [inline CHANGELOG.md:29]: Where's the VHDs?

**Review by palma21 — CHANGES_REQUESTED** [review]

**Comment by kaarthis** [comment]
https://github.com/Azure/AgentBaker/tree/master/vhdbuilder/release-notes no new VHDs yet. lets go ahead and publish first.

**Review by miwithro — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]

**Comment by kaarthis** [comment]
@palma21 pls approve and so we can merge it.

**Review by palma21 — APPROVED** [review]

**Pull request merged**